kin-recovery/README.md

```diff
@@ -135,21 +133,23 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 #### Step 4 - Backup or Restore
 Before you start using the Backup and Restore flows, you need to create a kinClient object.
+If you want to back up, you need the KinAccount object, which represents the account that you want to back up.
 ###### Example of how to create a kinClient object:
 
 ```java
 kinClient = new KinClient(context, Environment.TEST, "1acd")
 ...
 backupAndRestoreManager = new BackupAndRestoreManager(context);
```

Comment:
API change - add the req code params here and also documentation

Reply:
Already did it in a different PR which you will see as soon as i merge this one.